lib/systems/architectures.nix

```diff
@@ -55,6 +56,7 @@ rec {
     cascadelake    = [ "skylake-avx512" ] ++ inferiors.cannonlake;
     cooperlake     = [ "cascadelake"    ] ++ inferiors.cascadelake;
     tigerlake      = [ "icelake-server" ] ++ inferiors.icelake-server;
+    alderlake      = [ "syklake"        ] ++ inferiors.tigerlake;
```

Comment:
skylake?

Reply:
Sorry for the typo. It should be `[ "skylake" ] ++ inferiors.skylake`.

However I noticed that according to [gcc document](https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html), there exist instructions supported by all old cpus (from westmere to tigerlake) but not supported by alderlake, such as `CX16`. I am not an expert in CPU nor nixpkgs, should alderlake be a superior CPU of skylake in such conditions, or should alderlake be a superior CPU of none of old series?

Reply:
Typos often create bugs but that's the cool thing about code review: More people looking at the code before it's shipped, so these typos are more likely to be caught. Less pressure for the author to make things perfect too.